content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Server Software
+description: A detailed guide on which server software is right for you
+---
+
+import { Card, Cards } from "fumadocs-ui/components/card";
+
+# Introduction
+
+Everybody has different requirements when setting up their server. This guide will help you decide which server software is right for you.
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
+  because server softwares are usually distributed as a `.jar` file.
+</Callout>
+
+# Backend Servers
+
+Backend servers are the most common type of server software.
+There are an incomprehensible amount of backend servers available, however this section will cover the most recommended, stable and widely-used softwares.
+
+<Callout type="info" title="Note">
+  A "fork" is a copy of a software project that is modified in some way.\
+  They typically share the same codebase but have modified (or additional) features and optimizations.\
+  An example of "forks" include: Bukkit -> CraftBukkit -> Spigot -> Paper -> Pufferfish -> etc.
+</Callout>
+
+## Vanilla
+
+Vanilla is the official server software for Minecraft. As the name implies, it is the most basic server software available and has no modifications whatsoever.\
+It is ideal for those who wish to host a small server for a couple of friends or family, **without** altering any mechanics.\
+This server software is **not** recommended for larger servers.
+
+<Card href="https://jar.setup.md/download/vanilla/latest" title="Download">
```

Comment:
Why would we not link directly to Mojang for the download?

I would prefer not depending on Setup.md for our download links. If we can change them all to direct links to their respective primary sources, I think that would be better.

Reply:
This was resolved in one of Blstmo's commits.

Reply:
Nevermind, all of the links were changed to mcjars.

Reply:
[99a4258](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/99a4258a940a1f0933994604bf46580045a9de70) has moved all of the links to the official sites, **apart from Folia** - Should we keep the mcjars link or should we state that it requires manual compilation of the code?